src/lib/CircleLayer.svelte

```diff
@@ -24,6 +24,7 @@
   export let maxzoom: number | undefined = undefined;
   /** Set the cursor style to this value when the mouse is over the layer. */
   export let hoverCursor: string | undefined = undefined;
+  export let manageHoverState = false;
```

Comment:
As an aside, most of the layers plumb through many of the same boilerplate props. From my very vague understanding of the new Svelte 5 `$props` runes, it might become easier in the future to declare a TS interface for all the common props and reuse them more nicely. (Or maybe it's possible now.)

Reply:
Yes, I am hoping to do exactly that! I think it's sort of possible with Svelte 3/4 but doesn't work that great. Should be much easier when Svelte 5 comes around.